### PR TITLE
Serve contribute.json from the root of the repo.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

File: contribute.json [hunk 1]
```diff
@@ -11,10 +11,10 @@
         "mailing-list": "https://lists.mozilla.org/listinfo/dev-community-tools",
         "irc": "irc://irc.mozilla.org/#commtools",
         "irc-contacts": [
-            "giorgos",
             "nemo-yiannis",
             "tasos",
-            "williamr"
+            "pierros",
+            "comzeradd"
```
Comment (comzeradd) [inline contribute.json]: I've changed my irc nick to `nikos` :)

**Comment by johngian** [comment]
@akatsoulas r?

File: mozillians/humans/views.py [hunk 2]
```diff
@@ -14,5 +14,4 @@ def humans(request):
 @allow_public
 def contribute_view(request):
     """Generate a contribute.json"""
-    template = render(request, 'humans/contribute.json')
-    return HttpResponse(template, mimetype='application/json')
+    return serve(request, 'contribute.json', document_root=settings.ROOT)
```
Comment (akatsoulas) [inline mozillians/humans/views.py]: You can serve this directly from the appropriate urls.py and remove the view.

**Comment by akatsoulas** [comment]
r+wc :beetle:

File: mozillians/humans/views.py (hunk 2 above)
Comment (johngian) [inline mozillians/humans/views.py]: In our case we need to explicitly ``@allow_public`` to allow unauthenticated users to view ``contribute.json``.

Comment (glogiotatidis) [inline mozillians/humans/views.py]: tip: you can still do this without a view https://github.com/mozilla/mozillians/blob/master/mozillians/phonebook/urls.py#L48-L53

File: mozillians/humans/views.py [hunk 3]
```diff
@@ -14,5 +14,4 @@ def humans(request):
 @allow_public
 def contribute_view(request):
     """Generate a contribute.json"""
```
Comment (glogiotatidis) [inline mozillians/humans/views.py]: nit: we don't generate, we just serve

**Pull request merged**